Plugins/SpatialGDK/SpatialGDKEditorToolbar/Source/SpatialGDKEditorToolbar/Public/SpatialGDKEditorToolbarSettings.h

```diff
@@ -25,6 +25,14 @@ class USpatialGDKEditorToolbarSettings : public UObject
 	UPROPERTY(EditAnywhere, config, Category = "Configuration", meta = (ConfigRestartRequired = false, DisplayName = "Stop on Exit"))
 	bool bStopSpatialOnExit;
 
+	/** Path to your SpatialOS snapshot. */
+	UPROPERTY(EditAnywhere, config, Category = "Configuration", meta = (ConfigRestartRequired = false, DisplayName = "Snapshot path"))
+	FDirectoryPath SpatialOSSnapshotPath;
+
+	/** Name of your SpatialOS snapshot file. */
+	UPROPERTY(EditAnywhere, config, Category = "Configuration", meta = (ConfigRestartRequired = false, DisplayName = "Snapshot file name"))
```

Comment:
should this include the extension?

Reply:
if so we should document that as a requirement

Reply:
The extension is not required.